### PR TITLE
Fix icons that should not be flipped

### PR DESCRIPTION
**Pull request by huayunh**
#### Changes proposed in this Pull Request:
- Fix the icons that should not be flipped. Refer to https://github.com/pxblue/react-design-patterns/pull/30

#### Screenshots:
image
(only the log out icon is flipped, as it carries some directionality)

### CODE REVIEW

File: demos/storybook/stories/drawer/with-full-config.tsx [hunk 1]
```diff
@@ -218,7 +221,7 @@ export const withFullConfig = (context: DrawerStoryContext): StoryFnReactReturnT
             onClick: (): void => {
                 context.store.set({ selected: schedule });
             },
-            icon: <AirportShuttle />,
+            icon: <AirportShuttle style={getLeftToRightIconDirection()} />,
```
Comment (joebochill) [inline demos/storybook/stories/drawer/with-full-config.tsx:224]: This syntax reads cleaner...do you want to also use it for the 3 stories above for consistency?

File: demos/storybook/stories/user-menu/within-a-toolbar.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ export const withinToolbar = (): StoryFnReactReturnType => {
         title: {
             color: Colors.gray[600],
             fontSize: 12,
-            textAlign: 'right',
+            textAlign: getDirection() === 'rtl' ? 'left' : 'right',
```
Comment (joebochill) [inline demos/storybook/stories/user-menu/within-a-toolbar.tsx:20]: Is this necessary? Shouldn't the ILI be handling RTL correctly already?

Comment (huayunh) [reply demos/storybook/stories/user-menu/within-a-toolbar.tsx:20]: Yes, it is necessary. This gets rendered as an inline style. And we want to have a right aligned row

File: demos/storybook/stories/drawer/with-full-config.tsx (hunk 1 above)
Comment (huayunh) [reply demos/storybook/stories/drawer/with-full-config.tsx:224]: As long as it doesn’t hurt copy-ability of the code

File: demos/storybook/stories/user-menu/within-a-toolbar.tsx (hunk 2 above)
Comment (joebochill) [reply demos/storybook/stories/user-menu/within-a-toolbar.tsx:20]: Oh I see now, it's reversed (we want right aligned for LTR language)...got it.

**Review by joebochill — APPROVED** [review]

**Pull request merged**